src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -241,11 +245,59 @@ const DismissButton = styled(Button)`
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 `;
+const ExampleEventCause = styled(Panel)`
+  overflow: hidden;
+  pointer-events: none;
+`;
+
+const ExampleRibbon = styled('div')`
+  position: absolute;
+  right: 0px;
+
+  &:after {
+    display: block;
+    content: 'Example';
+    position: absolute;
+    top: 16px;
+    right: -24px;
+    text-transform: uppercase;
+    background: #e46187;
```

Comment:
Do we have a `theme` color for this?

Reply:
Didn't see anything. The `theme` pinks look a bit off for this